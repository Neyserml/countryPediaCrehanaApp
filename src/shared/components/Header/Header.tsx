import React from "react";
import styled from "styled-components";

const StayledHeader = styled.nav`
  & .nav-wrapper {
    background-color: #070e27 !important;
  }
`;

const Header: React.FC = () => {
  return (
    <StayledHeader>
      <nav className="nav-wrapper">
        <a href="#" className="brand-logo">
          Crehana
        </a>
      </nav>
    </StayledHeader>
  );
};

export default Header;
