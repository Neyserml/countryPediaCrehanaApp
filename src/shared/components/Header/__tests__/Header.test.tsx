import { render, screen } from "@testing-library/react";
import React from "react";
import Header from "../Header";

const renderComponent = () => {
  return render(<Header />);
};

describe("Header component", () => {
  it("should match snapshot in header page", () => {
    const component = renderComponent();
    expect(component.container).toMatchSnapshot();
  });

  it("should show title", () => {
    renderComponent();
    const title = screen.queryByText("Crehana");
    expect(title).not.toBeNull();
  });
});
