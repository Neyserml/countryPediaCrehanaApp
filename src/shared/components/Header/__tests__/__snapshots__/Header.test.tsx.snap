// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot in header page 1`] = `
<div>
  <nav
    class="sc-bdfBwQ dmBKaF"
  >
    <nav
      class="nav-wrapper"
    >
      <a
        class="brand-logo"
        href="#"
      >
        Crehana
      </a>
    </nav>
  </nav>
</div>
`;
